src/pymor/models/neural_network.py

```diff
@@ -76,6 +78,60 @@ def _compute_solution(self, mu=None, **kwargs):
 
             return U
 
+    class NeuralNetworkOutputModel(Model):
+        """Class for models of the output of stationary problems that use ANNs.
```

Comment:
I don't like the terminology very much: the `NeuralNetworkModel` can have outputs as well. However, I must admit that I have no better idea for the name at the moment.

Reply:
You are right, I totally agree, the name is not perfect. I will think about this a bit more. Maybe we can find a name that better reflects that this reductor just approximates the mapping from parameter to output. At the moment though, I don't have a handy phrase for that.